src/egon/data/datasets/etrago_setup.py

```diff
@@ -162,18 +162,18 @@ class EgonPfHvLink(Base):
     bus1 = Column(BigInteger)
     type = Column(Text)
     carrier = Column(Text)
-    efficiency = Column(Float(53))
-    p_nom = Column(Numeric)
-    p_nom_extendable = Column(Boolean)
-    p_nom_min = Column(Float(53))
-    p_nom_max = Column(Float(53))
+    efficiency = Column(Float(53), server_default="1.")
+    p_nom = Column(Numeric, server_default="0.")
+    p_nom_extendable = Column(Boolean, server_default="False")
+    p_nom_min = Column(Float(53), server_default="0.")
+    p_nom_max = Column(Float(53), server_default="inf")
     p_min_pu = Column(Float(53))
     p_max_pu = Column(Float(53))
```

Comment:
There are no default values for `p_min_pu` and `p_max_pu`. I'm wondering if that was intended or is missing.

Reply:
The variables without default values exist because there was no information related to their default value in pypsa documentation.

Reply:
The PyPSA documentation gives information about this default values (0 for p_min_pu and 1 for p_max_pu). I added these values in commit 0b51d2e13684f29198e58e5fabba0600db6fc31f